gm4_chairs/data/chairs/functions/kill.mcfunction

```diff
@@ -0,0 +1,7 @@
+# @s = @e[tag=gm4_chairs] and NOT inside stair blocks
+#execute at @s align xyz run tp @e[nbt={RootVehicle:{Entity:{id:"minecraft:pig"}}},dx=0,dz=0,dy=2] ~ ~.4 ~
+tp @s ~ ~.5 ~

```

Comment:
Why is the pig tp-ed?

Reply:
The pig is slightly inside the block under the stairs, so if the pig is killed the saddle item would glitch inside the block. it's much smoother if the pig is actually killed half a block up

Reply:
* As said in my other comments: I think killing the pig without drops and then summoning the saddle is better. This would reduce this file to 2 commands
* Wouldn't the block the pig is in be air, as it's no longer in a stair?

Reply:
Changed it to the 2 commands as you said, downside is that it now just appears instantly instead of going in a random direction as if it popped off